engine/src/flutter/lib/web_ui/lib/src/engine/canvaskit/canvaskit_api.dart

```diff
@@ -1790,10 +1790,10 @@ extension type SkParagraphBuilderNamespace(JSObject _) implements JSObject {
   );
 
   bool RequiresClientICU() {
-    if (!js_util.hasProperty(this, 'RequiresClientICU')) {
+    if (!hasProperty('RequiresClientICU'.toJS).toDart) {
       return false;
     }
-    return js_util.callMethod(this, 'RequiresClientICU', const <Object>[]) as bool;
+    return callMethod<JSBoolean>('RequiresClientICU'.toJS).toDart;
```

Comment:
I'd consider just adding the `external` method to the type and calling it that way.

Is Flutter doing this to explicitly avoid exposing a method that may or may not be there? If so, then feel free to ignore this and all similar suggestions.

Reply:
I think it's to handle when these bits don't exist 🤷